### DISCUSSION TITLE
[sound@cinnamon.org] Display input mute status

### DISCUSSION THREAD

**Discussion by sphh** [linuxmint/discussions#209] [Ideas]
Currently the sound applet does not show, if the microphone is muted or active. In the area of video conferences IMHO this is an important information.

If I understood the source for the sound applet correctly, it changes the icon depending, if a stream is played and/or the output is muted. This does not make it possible to show additional information, e.g. if the input is muted.

I want to suggest to use badges to show the states of the input and output streams on top of the main icon, e.g. (obviously this could be improved by using the right icons for the badges):

![sound-applet-mock-up](https://github.com/linuxmint/discussions/assets/4856462/1cb59948-72af-4834-b27c-d73e9a7b3011)

What do you think?

Related:
- PR #11914 Adds an option to always show the input switch in the sound applet, so that the microphone can be muted before e.g. starting the conferencing program.
- PR #11913 Adds a shift-middle click action to the sound applet: Up to now it was possible to bind one action to a middle click onto the sound icon. This might not be enough, because you might want to play/pause the current stream _and_ also be able to mute the microphone with a mouse click. Now you can configure the middle-click and the shift-middle-click to two different actions.

**Comment by mohicanb** [comment]
Yes and I would add : 
include the **microphone volume control (with mute/unmute + a slide bar) in the sound applet**.

I am using both Cinnamon and Xfce DE and this feature is really **the one that makes me stay in Xfce**. I really feel it's missing in Cinnamon.
Thank you

**Comment by claudiux** [comment]
https://github.com/linuxmint/cinnamon-spices-applets/pull/5574

You can now try the version 6.3.0 of the `sound150@claudiux` applet: https://cinnamon-spices.linuxmint.com/applets/view/306

**Reply by mohicanb** [reply]
@claudiux Nice. Do you think you can add a slider for the mic volume ?

**Reply by claudiux** [reply]
In `sound@cinnamon.org` the mic volume slider is in the context menu (right-click); in `sound150@claudiux`, it's in menu (left-click).

**Comment by sphh** [comment]
It would be great, if your great work from `sound150@claudiux` would trickle back to the standard sound applet. Do you plan to transfer it?
